Class/Practise 3.2/include/Color.hpp

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+class Color {
+public:
+    Color(int r, int g, int b);
+    Color magicColor() const;
```

Comment:
getMagicColor, calculateMagicColor or generateMagicColor
also it is better to add [[nodiscard]] - we don't expect that someone who called this method will be interested to ignore the result

Reply:
Refactored the code in [8c51226](https://github.com/YaroslavKSE/Professinal-Programming/pull/9/commits/8c5122622175b05897c9517073d7a3788bc01321)